understanding/21/non-text-contrast.html

```diff
@@ -210,58 +241,109 @@ <h4>Gradients</h4>
+						<figure>
+							<img src="img/graphics-contrast_pie-chart_fail.png" alt="Failing pie chart" />
+							<figcaption>
+								<p><strong>Fail:</strong> The pie chart has labels for each slice (so passes 1.4.1 Use of Color), but in order to understand the proportions of the slices you must discern the edges of the slices (the graphical objects conveying essential information), and the contrast between the slices is not over 3:1. </p>
+							</figcaption>
+						</figure>
+						<figure>
+							<img src="img/graphics-contrast_pie-chart_na.png" alt="Not applicable pie chart" />
+							<figcaption>
+								<p><strong>Not applicable:</strong> The pie chart has visible labels <em>and</em> values that convey equivalent information to the graphical objects (the pie slices). </p>
+							</figcaption>
+						</figure>
+						<figure>
+							<img src="img/graphics-contrast_pie-chart_pass.png" alt="Passing pie chart" />
+							<figcaption>
+								<p><strong>Pass:</strong> The pie chart has visible labels, and sufficient contrast around and between the slices of the pie chart (the graphical objects). A darker border has been added around the yellow slice in order to achieve the contrast level.</p>
+							</figcaption>
+						</figure>
+					</section>
+					<section class="example">
+						<h4>Infographics</h4>
+						<figure>
+							<img src="img/infographic-fail.png" alt="An infographic showing lightly coloured circles of various sizes to indicate the size of five different social networks" />
+							<figcaption>
+								<p><strong>Fail:</strong> Discerning the circles is required to understand the size of network and discerning the icons in each circle is required to identify which network it shows.</p>
+							</figcaption>
+						</figure>
+						<p>The graphical objects are the circles (measured against the background) and the icons in each circle (measured against the circle's background).</p>
+						<figure>
+							<img src="img/infographic-pass.png" alt="The same infographic with contrasting text, dark borders around the circles, and contrasting icons." />
+							<figcaption>
+								<p><strong>Pass:</strong> The circles have contrasting borders and the icons are a contrasting dark colour against the light circle backgrounds.</p>
+							</figcaption>
+						</figure>
+						<p>There are many possible solutions to ensuring contrast, the example shows the use of borders. Other techniques are to use darker colours for the circle backgrounds, or to add text labels &amp; values for each item.</p>
+					</section>
+				</section><!-- end graphic-object-examples -->
+
+			</section><!-- end graphical-objects -->
+
+			<section id="testing-principles">
+					<h3>Testing Principles</h3>
```

Comment:
I'm not entirely sold on this section. I can live with it, but think that this understanding document is already long and this moves into the techniques territory, so I would like to figure out how to integrate it into the structure better or leave it for techniques, sooner or later.

Reply:
If someone can think of a better way of doing that, happy to adjust. The main things is that the SC covers two quite different elements on the page, each of which have multiple ways to meet the SC, so no structure of and/or for the techniques will address this.